### DISCUSSION TITLE
Problem downloading (exporting to CSV) big pixel intensities file

### DISCUSSION THREAD

**Discussion by trungfupanda** [metaspace2020/metaspace#1595] [Q&A] [answered]
I don't know if this is just me: On the website, when I set the FDR 20%, I can download my dataset's pixel intensities (around 100-200 MB). But if I set to FDR 50%, It loads for a while but no file is prompted to download in the end. 

Also, is there any way to download pixel intensities file through Python Client ? 
Thanks!

**Comment by Bisho2122** [comment, marked as answer]
Regarding downloading pixel intensities using API, if you're familiar with AnnData format, you can use metaspace-converter https://github.com/metaspace2020/metaspace-converter and just enter your dataset Id and FDR as shown in the short tutorial. If it doesn't work, happy to provide a custom script :)
